packages/pigment-react/tests/css/css.test.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'node:path';
+import { expect } from 'chai';
+import { runTransformation } from '../testUtils';
+
+const theme = {
+  palette: {
+    primary: {
+      main: 'red',
+    },
+  },
+  size: {
+    font: {
+      h1: '3rem',
+    },
+  },
+  components: {
+    MuiSlider: {
+      styleOverrides: {
+        rail: {
+          fontSize: '3rem',
+        },
+      },
+    },
+  },
+};
+
+const CUSTOM_ERROR =
+  'The file contents have changed. Run "test:update" command to update the file if this is expected.';
+
+describe('Pigment CSS - css', () => {
+  it('basics', async () => {
+    const { output, fixture } = await runTransformation(
+      path.join(__dirname, 'fixtures/css.input.js'),
+      {
+        themeArgs: {
+          theme,
+        },
+      },
+    );
+
+    expect(output.js, CUSTOM_ERROR).to.equal(fixture.js);
+    expect(output.css, CUSTOM_ERROR).to.equal(fixture.css);
```

Comment:
Maybe create a wrapper util as well that just takes the path and theme and calls `runTransformation` as well as `expect`. This way, there won't be any duplicated logic for similar tests.

Reply:
I reexport the `expect` from the test utils instead because `it(…)` should be used in the test files. It reduces the custom error duplicates. I think it's good enough.